### ISSUE TITLE
Fix missing alt text

### ISSUE THREAD

**Issue by riccardoporreca** [miraisolutions/techguides#24]
We currently ignore missing img alt text when using HTMLProofer, due to a former issue that is probably resolved by now: see https://github.com/miraisolutions/techguides/issues/13#issuecomment-810083276

Meaningful alt texts should included and the `--empty-alt-ignore` removed from the Actions WF
